cmd/github-labels/README.md

```diff
@@ -0,0 +1,75 @@
+* [Overview](#overview)
+* [Generating the combined labels database](#generating-the-combined-labels-database)
+* [Checking and summarising the labels database](#checking-and-summarising-the-labels-database)
+    * [Show labels](#show-labels)
+    * [Show categories](#show-categories)
+    * [Check only](#check-only)
+    * [Full details](#full-details)
+
+# Overview
+
+The Kata Project uses a number of GitHub repositories. To allow issues and PRs
+to be handled consistently between repositories a standard set of issue labels
+are used. These labels are stored in YAML format in the master [labels
+database template](labels.yaml.in). This file is human-readable and
```

Comment:
Lines 14-16 suggested rewrite:
... This file is human-readable, machine-readable, and self-describing (see the file for the introductory description).